core/src/forwarding_stage.rs

```diff
@@ -5,6 +5,7 @@ use {
     crate::next_leader::next_leaders,
     agave_banking_stage_ingress_types::BankingPacketBatch,
     agave_transaction_view::transaction_view::SanitizedTransactionView,
+    async_trait::async_trait,
```

Comment:
async traits are stable no?

Reply:
not really, you cannot do something like:

```rust
trait MyTrait {
    async fn do_stuff();  // Not allowed in stable Rust
}
```

Reply:
Of course you can https://play.rust-lang.org/?version=stable&mode=debug&edition=2024

Reply:
Yes, you are right. I need to do this upgrade in separate PR for tpu-client-next, SendTransactionService and this FollowingStage. 

https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=0be513f6c2561889a4e9450c4bf65ce3

Reply:
This requires changes in the tpu-client-next and rpc, so will address in a separate PR.